python/demo/mixed-elasticity-sc/static-condensation-elasticity.py

```diff
@@ -60,19 +61,19 @@ def left(x):
 
 # Locate all facets at the free end and assign them value 1
 free_end_facets = locate_entities_boundary(mesh, 1, free_end)
-mt = dolfinx.mesh.MeshTags(mesh, 1, free_end_facets, 1)
```

Comment:
Should we use `mesh.topology.dim-1` here instead of 1?
I’ve found that most people on discourse does not understand the idea of the dimension of the entity.